bamboo-specs/bamboo.yaml

```diff
@@ -269,6 +269,10 @@ variables:
   ORCA_VERSION: 4.0.0
   RELEASE_FLAG: 'false'
   REPORT_BUILD_STATUS: 'true'
+  SECRET_GITHUB_EMAIL: BAMSCRT@0@0@HYUkicwfEqt17JZaZuGYmtmRHrnDySiT+kQkBQa7/Rg7IQgl1lUvbncMUu0Wx2k1ayHojNkh2TdOlF0kJ5tw6Q== #encrypted
```

Comment:
Did we determine that we need to add these to the release docs?

Reply:
Yes I did add them to the release docs already but didn't provide the actual values as they are sensitive.